<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePreferenceWidget</class>
 <widget class="QWidget" name="UpdatePreferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current version</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentVersionLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Check period</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkOnStartupEdit">
        <property name="text">
         <string>Check on startup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Check interval</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="checkIntervalEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> minute</string>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sources</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="sourcesList"/>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolButton" name="addSourceButton">
           <property name="toolTip">
            <string>Add</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/share/images/add.png</normaloff>:/share/images/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="editSourceButton">
           <property name="toolTip">
            <string>Edit</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/share/images/edit.png</normaloff>:/share/images/edit.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeSourceButton">
           <property name="toolTip">
            <string>Remove</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/share/images/remove.png</normaloff>:/share/images/remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveUpButton">
           <property name="toolTip">
            <string>Move up</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Move up</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/share/images/uparrow.png</normaloff>:/share/images/uparrow.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moveDownButton">
           <property name="toolTip">
            <string>Move down</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Move down</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/share/images/downarrow.png</normaloff>:/share/images/downarrow.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
